// funweb-functions/library.js
exports.printMsg = function(){
  console.log('printMsg din funweb-funcgions module');
}

exports.trimEnemy = function(enemy){
  var trimmed = {id: '', score: -1};
  trimmed.id = enemy.playerData.id;
  trimmed.score = enemy.score;
  return trimmed;
}

exports.trimRoomForJoinRoom = function(room){
  var trimmed = {
    number: room.number,
    name: 'room' + room.number
  }
  return trimmed;
}


// exports.isRoomFull = function(room){ /////////////// Este camera plina  ?
//   if(room.players.length<2) {
//     return false;
//   }
//   else{
//     return true;
//   }
// }


exports.isRoomEmpty = function(room){ /////////////// Exista jucatori in camera ?
  if(room.players.length>0) {
    return false;
  }
  else{
    return true;
  }
}

exports.socketsExistInRoom = function(room){
  for(var i = 0; i < room.players.length; i++){
    if( room.players[i].socket != null) return true;
  }
  return false;
}

exports.matchRoomUser = function(room, user){  ////////////  Algoritm de matching  ROOM - USER
  return true;
}
